service_management/azure/test/integration/storage_management/storage_management_test.rb

```diff
@@ -16,170 +16,170 @@
-  end
+    it 'get storage account properties' do
+      storage = subject.get_storage_account_properties(storage_name)
+      storage.name.must_equal storage_name
+      storage.label.must_equal 'storagelabel'
+      storage.account_type.must_equal 'Standard_GRS'
+    end
 
-  it 'get storage account keys' do
-    storage_name = StorageName
-    storage_keys1 = subject.get_storage_account_keys(storage_name)
-    storage_keys1.primary_key.wont_be_nil
-    storage_keys1.secondary_key.wont_be_nil
-  end
+    it 'regenerate storage account keys' do
+      storage_keys1 = subject.regenerate_storage_account_keys(storage_name)
+      storage_keys1.primary_key.wont_be_nil
+      storage_keys1.secondary_key.wont_be_nil
+      storage_keys2 = subject.regenerate_storage_account_keys(storage_name, 'secondary')
+      storage_keys1.primary_key.must_equal storage_keys2.primary_key
+      storage_keys1.secondary_key.wont_equal storage_keys2.secondary_key
+    end
 
-  it 'get storage account properties error' do
-    storage_name = 'invalidstorage'
-    exception = assert_raises(RuntimeError) do
-      subject.get_storage_account_properties(storage_name)
+    it 'get storage account keys' do
+      storage_keys1 = subject.get_storage_account_keys(storage_name)
+      storage_keys1.primary_key.wont_be_nil
+      storage_keys1.secondary_key.wont_be_nil
     end
-    assert_match(/The storage account 'invalidstorage' was not found/, exception.message)
-  end
 
-  it 'create storage account with invalid storage name' do
-    options = {
-      affinity_group_name: 'affinitygrouptest',
-      label: 'storagelabel',
-      description: 'This is a storage account',
-      geo_replication_enabled: 'false'
-    }
-    storage_name = 'ba'
-    exception = assert_raises(RuntimeError) do
-      subject.create_storage_account(storage_name, options)
+    it 'update existing storage account' do
+      options = {
+          label: 'labelchanged',
+          description: 'description changed',
+          account_type: 'Standard_LRS'
+      }
+      subject.update_storage_account(storage_name, options)
+      storage = subject.get_storage_account_properties(storage_name)
+      storage.name.must_equal storage_name
+      storage.label.must_equal 'labelchanged'
+      storage.account_type.must_equal 'Standard_LRS'
+      opts[:account_type] = 'Standard_GRS'
     end
-    assert_match(/Storage account names must be between 3 and 24/, exception.message)
   end
 
-  it 'create storage account with invalid location' do
-    options = {
-      location: 'West1 US',
-      label: 'storagelabel',
-      description: 'This is a storage account',
-      geo_replication_enabled: 'false'
-    }
-    exception = assert_raises(RuntimeError) do
-      subject.create_storage_account(storage_name, options)
+  describe 'storage management errors' do
+
+    before do
+      Azure::Loggerx.expects(:puts).returns(nil).at_least(0)
```

Comment:
Is it possible to put this line on before block of "describe Azure::StorageManagementService do", so that we don't need it in each sub describe context?